### DISCUSSION TITLE
verify a raven server is running using the ravendb client?

### DISCUSSION THREAD

**Discussion by seanrockster** [ravendb/ravendb#16169] [General]
Hi

How can I verify a raven server is running using the ravendb client? I'm sure this situation shouldn't occur, because in my previous cases if the urls where not correct, an exception would be thrown. However I'm starting a new project using RavenDb.Client v5.4.103, no matter what url i use, completely made up, non-existent, and I can create a store, call initialize, open a session and call .SaveChanges() on the session. 

![J3Tf2zhiSG](https://user-images.githubusercontent.com/2518884/229453646-99ca28b5-c3f4-4aa1-9b4b-bf0a36635c92.png)

**Comment by ayende** [comment]
Does this complete without an error?!
What do you get here?

Note that creating a document store or opening a session do *not* send any calls to the server.
The easiest way to verify the server is up is calling it, trying to load a document (including non existant one)

**Reply by seanrockster** [reply]
Yes, I can call .SaveChanges() and no error is thrown but I'm not storing anything. I'll try to save something and see if I get an exception.

**Reply by ayende** [reply]
Oh, sure, you aren't saving anything so we never go to the server :-)
You do a load, and it will work

**Comment by seanrockster** [comment]
No error on .SaveChanges(). Bizarre. 

![vSZ4YNpfkn](https://user-images.githubusercontent.com/2518884/229501035-afb99c47-48ef-40a3-a22c-80398ef9ffc5.png)

**Reply by seanrockster** [reply]
Ah, nevermind, it throw when leaving the using statement. Sorted.
